magento/pwa-studio
# [PWA-577] Mini cart product options (color/size) should be separate lines.

## Description

URL - https://develop.pwa-venia.com/

Steps -
1. Add configurable product to cart.
2. Observe mini cart product UI.

Expected - Product option should not be wrapped.
Actual - Product Options wrapping.

## Related Issue





* [[PWA-577](https://jira.corp.magento.com/browse/PWA-577)]

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Add Configurable Product from our sample data to your cart
2. Open MiniCart, verify that options are now on new lines

## Screenshots / Screen Captures (if appropriate)
<img width="241" alt="Screen Shot 2020-05-12 at 12 34 04 PM" src="https://user-images.githubusercontent.com/462953/81726373-e47c5400-944c-11ea-8e11-990399a117e3.png">

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
  * No test included given this is a presentational fix
